draft-irtf-cfrg-hpke.md

```diff
@@ -1090,10 +1165,20 @@ AuthPSK mode if the pre-shared key and the recipient private key `skR` are
 both compromised. NaCl's `box` interface {{NaCl}} has the same issue. At
 the same time, this enables repudiability.
 
+Even though it is possible to construct an authenticated
+KEM that is resistant against key-compromise impersonation, a key-compromise
+impersonation stays generally possible on the level of HPKE, as shown by {{ABHKLR20}}.
+That is because the KEM ciphertext is not bound to the HPKE message.
+Thus, an adversary who knows a recipients private key, can decapsulate an
+observed KEM ciphertext, compute the key schedule, and encrypt an arbitrary
+message that the recipient will accept as coming from the original sender.
+Mitigating this issue would require fundamental changes, which are
+out-of-scope of this document.
+
 Applications that require resistance against key-compromise impersonation
 SHOULD take extra steps to prevent this attack. One possibility is to
-produce a digital signature over the Auth and AuthPSK `enc` output using
```

Comment:
Signing only the KEM ciphertext would not solve the problem pointed out by the generic attack from ABHKLR20.